Documentation/BuildInstructionsMacOS.md

```diff
@@ -34,3 +34,4 @@ Notes:
 
```

Comment:
Regarding the commit message: we generally don't put line breaks in the middle of a paragraph.

So do something like this instead:
```
This PR includes information that highlights the importance of
updating Xcode on MacOS. I ran into problems building serenity
on MacOS because I had Xcode installed but not updated in a while.
This triggered seemingly unrelated errors that were easily solved
by updating Xcode.
```
Also there is no need to put opinions into commit messages.